tests/functions/functions.test.js

```diff
@@ -6,7 +6,8 @@ const { expect } = require('chai');
 const { expect: jestExpect, it } = require('@jest/globals');
 
 const { getTmpDirPath, replaceTextInFile } = require('../utils/fs');
-const { getServiceName, sleep, serverlessDeploy, serverlessRemove} = require('../utils/misc');
```

Comment:
IDE was signalling a text convention error, so I had to break it down into two lines